makefile

```diff
@@ -13,6 +13,11 @@ else
 	K := $(foreach exec,$(EXECUTABLES),$(if $(shell which $(exec)),some string,$(error "No $(exec) in PATH")))
 endif
 
+# Docker buildkit builds work in parallel, lowering build times. Outputs are compatable.
+# Requires docker 18.09 to be effective (and is ignored on newer docker)
```

Comment:
This comment is a little confusing.  So this is enabled past Docker 18.09? And ignored for anything newer?

Reply:
Yes. Docker 18.09 added support for docker buildkit mode. Earlier versions will ignore this flag.

There is a noticeable difference in output between the two modes, so the comment is meant to help folks grok why it might look different.

Reply:
We worked out a better comment in slack.